dspace-api/src/main/java/org/dspace/app/itemimport/ItemImportServiceImpl.java

```diff
@@ -1518,28 +1676,62 @@ protected void processOptions(Context c, Item myItem, List<String> options)
             if (notfound && !isTest) {
                 // this should never happen
                 System.out.println("\tdefault permissions set for "
-                                       + bitstreamName);
+                    + bitstreamName);
             } else if (!isTest) {
                 if (permissionsExist) {
                     if (myGroup == null) {
                         System.out.println("\t" + groupName
-                                               + " not found, permissions set to default");
+                            + " not found, permissions set to default");
                     } else if (actionID == -1) {
                         System.out
                             .println("\tinvalid permissions flag, permissions set to default");
                     } else {
                         System.out.println("\tSetting special permissions for "
-                                               + bitstreamName);
+                            + bitstreamName);
                         setPermission(c, myGroup, actionID, bs);
                     }
                 }
 
                 if (descriptionExists) {
                     System.out.println("\tSetting description for "
-                                           + bitstreamName);
+                        + bitstreamName);
                     bs.setDescription(c, thisDescription);
                     bitstreamService.update(c, bs);
```

Comment:
now that multiple metadata of the bitstream can be eventually set it would be more efficient if we store in a flag the need to perform a `bitstreamService.update()` a do it just one time for each bitstream